mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -108,7 +112,13 @@ public void configureInputJobCredentials(TableDesc tableDesc, Map<String, String
   public void configureJobConf(TableDesc tableDesc, JobConf jobConf) {
     if (tableDesc != null && tableDesc.getProperties() != null &&
         tableDesc.getProperties().get(WRITE_KEY) != null) {
+      Preconditions.checkArgument(!tableDesc.getTableName().contains(".."),
```

Comment:
nit: can we put the `".."` string in a constant too? seems to be used in a few places

Reply:
Good point. Fixed